packages/mephisto-worker-experience/src/index.css

```diff
@@ -0,0 +1,207 @@
+/* Headless Classes */
+.headless-mephisto-worker-experience-tips__tips-list {
```

Comment:
unfortunately we'll have to rename these styles to `mephisto-worker-addons`. apologies for the thrash here, should be a simple find/replace